### PR TITLE
Update windows and osx guide w/ commit 5766ce6 change

### PR DESCRIPTION
**Pull request by gratefulcheddar**
Fix the typo in the OSX and Windows getting started guide for the walletpassphrase command

### CODE REVIEW

File: docs/getting-started/user-guides/osx.md [hunk 1]
```diff
@@ -249,7 +249,7 @@ Decred. Open a third terminal and go to your Decred directory. Type in
 (once again using your values for the square brackets):
 
 ```no-highlight
-~/Decred/$ ./dcrctl -u [daemon username] -P [daemon password] --wallet walletpassphrase [wallet password] 0
+~/Decred/$ ./dcrctl -u [wallet username] -P [wallet password] --wallet walletpassphrase [wallet password] 0
 ```
 
 What we are doing here is authenticating with the daemon and using the
```
Comment (jcvernaleo) [inline docs/getting-started/user-guides/osx.md]: I think with the above change this should say 'authenticating with the wallet', not the daemon here.

File: docs/getting-started/user-guides/windows.md [hunk 2]
```diff
@@ -250,7 +250,7 @@ Decred. Open a third command prompt in your Decred folder. Type in
 (once again using your values for the square brackets):
 
 ```no-highlight
-C:\Decred> dcrctl -u [daemon username] -P [daemon password] --wallet walletpassphrase [wallet password] 0
+C:\Decred> dcrctl -u [wallet username] -P [wallet password] --wallet walletpassphrase [wallet password] 0
 ```
 
 What we are doing here is authenticating with the daemon and using the
```
Comment (jcvernaleo) [inline docs/getting-started/user-guides/windows.md]: I think with the above change this should say 'authenticating with the wallet', not the daemon here.

**Comment by jcvernaleo** [comment]
@gratefulcheddar Thanks for the PR!  Just a couple minor things needed before I can get it in.

**Comment by gratefulcheddar** [comment]
@jcvernaleo That should fix it! Hope I did everything correctly, still new to git/github.

**Comment by jcvernaleo** [comment]
Looks like you took care of all the issues I had.  Thanks!

**Pull request merged**